app/controllers/users_controller.rb

```diff
@@ -1,6 +1,6 @@
 class UsersController < ApplicationController
   before_action :logged_in_user, only: %i(show new create)
-  before_action :load_user, only: %i(index new create)
+  before_action :load_user, except: %i(edit update destroy show)
```

Comment:
for line 3,
if i use only: i could not click sign up. it will show flash error (user not found). 
if i use except: i can go to sign up page

![image](https://user-images.githubusercontent.com/68294917/91538250-207aec80-e94a-11ea-86df-97d9d122f3af.png)